OpenRA.Mods.Common/Activities/Air/ReturnToBase.cs

```diff
@@ -147,6 +147,8 @@ public override Activity Tick(Actor self)
 				}
 			}
 
+			self.SetTargetLine(Target.FromActor(dest), Color.Green);
```

Comment:
Our conventions are that `SetTargetLine()` with `display: true` must *only* be called from `ResolveOrder`. This prevents improper visual feedback popping up when activities are queued from other activities.

This could stay, but needs to use the three argument version, and we will need to keep a call in `Aircraft` that gives the visual feedback.  The logic here will be simpler once #2598 is resolved.

Reply:
Reverted the target line changes as discussed on IRC.